test/integration/index.js

```diff
@@ -772,7 +816,7 @@ describe('bootstrap()', () => {
 
   });
 
-  describe('with ga-tag', () => {
+  describe('with locals', () => {
```

Comment:
I have nested the ga-tag code so we can make use of the same set up for nonce values in locals